astropy/coordinates/angles/formats.py

```diff
@@ -399,32 +399,32 @@ def parse_angle(angle, unit=None, debug=False):
     return _AngleParser().parse(angle, unit, debug=debug)
 
 
+def _angle_to_sexagesimal(a, /):
```

Comment:
This might be a better name:
```suggestion
def _decimal_to_sexagesimal(a, /):
```

Reply:
thanks, I'll use that in my next push !